docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+
+`options.order`: Required array of strings.
+                 An array of the names of `fields` or `helpers`, in order, that are to be
+                 formatted within the row. It may contain any field, but only helpers of the type "static" and "group".
+
+The size of this array must equal the `options.colSize` value of the table to which the row belongs.
+
+### 4.1.4: Table
+
+A helper of type "table" denotes a list of "row" helpers that are displayed together in a tabular
+form.
+
+A table contains only "row" helpers whose `options.order` array must be of the same size.
+
+```js
+{
+    "schema" : {
+        ...
+        "helpers" : {
+            "nameOfHelper" : {
+                "type": "table",
+                "description": {
+                    ...
+                },
+                "options": {
+                    "colSize" : int,
+                    "rowSize" : int,
+                    "order": [
+                        string,
+                        string,
+                        ...
+                    ],
+                    "showColHeaders" : boolean,
+                    "showDesc" : boolean,
+                    "showAsFooter": boolean,
+                },
+                "rules": {
+                    ...
+                }
+        ...
+}
+```
+`options.colSize` : Required int.
+                    The number of columns in the table. 
+
+`options.rowSize` : Required int.
+                    The number of rows in the table.
+
+`options.order`: Required array of strings.
+                 An array of the names of `row` helpers, in order, that are to be formatted
+                 within this table.
+
+The size of this array must equal `options.rowSize`.
+
+At each index (or column) of every "row" element, the `helper` or `field` that is
+referenced inside that index has to have its `description` match. This description
+will be rendered as the column header if `options.showColHeaders` is true.
+
+The only exception is that a "static" helper will never have a corresponding header, 
+as its `description` is the static text itself. A column of static elements can have
+an empty column header. This is true for only static elements.
+
+`options.showColHeaders`: Boolean. Default is true.
+                          True or false that the table's column headers will be displayed as taken
+                          from each column's fields' matching `description`.
+
+`options.showDesc`: Boolean. Default is true.
+                    True or false that the table's `description` is displayed.
+
+`options.showAsFooter`: Boolean. Default is false.
+                        True or false that the description is displayed as a footer, instead of a
+                        header.
+
+### 4.1.5: Section
+
+A helper of type "section" is a vertical grouping of elements whose `rules` can affect its children's rules.
```

Comment:
I'm not sure what you mean by "affect its children's rules".. affect them how?

Reply:
if a section's `requireIf` evaluates to true, then all the helper/fields in the section will also be required a.k.a their `requireIf` becomes true